fastly/waf_active_rule.go

```diff
@@ -17,12 +17,14 @@ var WAFActiveRuleType = reflect.TypeOf(new(WAFActiveRule))
 
 // WAFActiveRule is the information about a WAF active rule object.
 type WAFActiveRule struct {
-	ID        string     `jsonapi:"primary,waf_active_rule,omitempty"`
-	Status    string     `jsonapi:"attr,status,omitempty"`
-	ModSecID  int        `jsonapi:"attr,modsec_rule_id,omitempty"`
-	Revision  int        `jsonapi:"attr,revision,omitempty"`
-	CreatedAt *time.Time `jsonapi:"attr,created_at,iso8601,omitempty"`
-	UpdatedAt *time.Time `jsonapi:"attr,updated_at,iso8601,omitempty"`
+	ID             string     `jsonapi:"primary,waf_active_rule,omitempty"`
+	Status         string     `jsonapi:"attr,status,omitempty"`
+	ModSecID       int        `jsonapi:"attr,modsec_rule_id,omitempty"`
+	Revision       int        `jsonapi:"attr,revision,omitempty"`
+	CreatedAt      *time.Time `jsonapi:"attr,created_at,iso8601,omitempty"`
+	UpdatedAt      *time.Time `jsonapi:"attr,updated_at,iso8601,omitempty"`
+	Outdated       bool       `jsonapi:"attr,outdated,omitempty"`
+	LatestRevision int        `jsonapi:"attr,latest_revision,omitempty"`
```

Comment:
Metadata at the bottom:

```suggestion
	ID             string     `jsonapi:"primary,waf_active_rule,omitempty"`
	Status         string     `jsonapi:"attr,status,omitempty"`
	ModSecID       int        `jsonapi:"attr,modsec_rule_id,omitempty"`
	Revision       int        `jsonapi:"attr,revision,omitempty"`
	Outdated       bool       `jsonapi:"attr,outdated,omitempty"`
	LatestRevision int        `jsonapi:"attr,latest_revision,omitempty"`
	CreatedAt      *time.Time `jsonapi:"attr,created_at,iso8601,omitempty"`
	UpdatedAt      *time.Time `jsonapi:"attr,updated_at,iso8601,omitempty"`
```